### PR TITLE
ci(root): add pull request template LS-1697

### PR DESCRIPTION
**Pull request by remarkablemark**
## Jira

[Add pull request template to design-system](https://littlespoon.atlassian.net/browse/LS-1697)

## Motivation

ci(root): add pull request template

## Current Behavior

No pull request template

## New Behavior

Pull request template

## Checklist

- [x] [Conventional Commits](https://www.conventionalcommits.org/)

### CODE REVIEW

**Review by eeeeean — DISMISSED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+## Jira
+
+<!--
+[](https://littlespoon.atlassian.net/browse/LS-)
+-->
+
+## Motivation
```
[the remark this answers is not in the document]
Comment (remarkablemark) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: Not mandatory, should I add a comment to note?

[the remark this answers is not in the document]
Comment (eddkmm) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: Sounds good to me. We could add an `[Optional]` tag to them

[the remark this answers is not in the document]
Comment (remarkablemark) [reply .github/PULL_REQUEST_TEMPLATE.md:7]: I'll add a comment to the top

[the remark this answers is not in the document]
Comment (remarkablemark) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: e7ae204

**Review by eddkmm — APPROVED** [review]

**Pull request merged**